src/react-components/layout/Footer.scss

```diff
@@ -78,6 +79,7 @@ footer {
 
   @media(min-width: theme.$breakpoint-lg) {
     margin-left: 18px;
+    height: 41px;
```

Comment:
Use "ems"?

Reply:
After testing on HMC and HC, I think we might be ok with keeping as is but will be looking to make changes to the footer in a future PR based on design recommendations.